server/src/main/java/jake2/server/SV_USER.java

```diff
@@ -147,9 +147,9 @@ private static void SV_Configstrings_f(List<String> args, GameImportsImpl gameIm
         final String nextCmd;
 
         if (start == Defines.MAX_CONFIGSTRINGS) {
-            nextCmd = "cmd baselines " + gameImports.spawncount + " 0\n";
+            nextCmd = "cmd " + StringCmdMessage.BASELINES + " " + gameImports.spawncount + " 0\n";
```

Comment:
easier to track usages this way

Reply:
Why not use the string templating?
```suggestion
            nextCmd = String.format("cmd %s %s 0\n", StringCmdMessage.BASELINES, gameImports.spawncount);
```

Reply:
done

Reply:
missing kotlin strings :(